h2o-py/h2o/explanation/_explain.py

```diff
@@ -1336,11 +1355,16 @@ def ice_plot(
                 nbins=20 if not is_factor else 1 + frame[column].nlevels()[0]
             )[0]
         )
+        if show_logodds:
+            logodds = np.log(tmp["mean_response"] / (1 - tmp["mean_response"]))
         if is_factor:
-            plt.scatter(factor_map(tmp.get(encoded_col)), tmp["mean_response"], color="k",
+            plt.scatter(factor_map(tmp.get(encoded_col)),
+                        tmp["mean_response"] if not show_logodds else logodds, color="k",
                         label="Partial Dependence")
         else:
-            plt.plot(tmp[encoded_col], tmp["mean_response"], color="k", linestyle="dashed",
+            plt.plot(tmp[encoded_col],
+                     tmp["mean_response"] if not show_logodds else logodds, color="k",
```

Comment:
nit/style/optional: I'd have a tendency to extract this condition, to align with the DRY principle 🙂  E.g.
```python
response = tmp["mean_response"]
if show_logodds:
    response = np.log(tmp["mean_response"] / (1 - tmp["mean_response"]))
```

Reply:
f3437b8